modules/skybrowser/skybrowsermodule_lua.inl

```diff
@@ -0,0 +1,589 @@
+    std::string nameTarget = "Sky Target " + std::to_string(noOfPairs);
+    std::string idBrowser = "SkyBrowser" + std::to_string(noOfPairs);
+    std::string idTarget = "SkyTarget" + std::to_string(noOfPairs);
+    glm::vec3 positionBrowser = { -1.0f, -0.5f, -2.1f };
+    std::string guiPath = "/SkyBrowser";
+    std::string url = "https://data.openspaceproject.com/dist/skybrowser/page/";
+
+    const std::string browser = "{"
+        "Identifier = '" + idBrowser + "',"
+        "Type = 'ScreenSpaceSkyBrowser',"
+        "Name = '" + nameBrowser + "',"
+        "Url = '" + url + "',"
+        "FaceCamera = false,"
+        "CartesianPosition = " + ghoul::to_string(positionBrowser) +
+        "}";
+    const std::string target = "{"
+        "Identifier = '" + idTarget + "',"
+        "Type = 'ScreenSpaceSkyTarget',"
+        "Name = '" + nameTarget + "',"
+        "FaceCamera = false,"
+        "}";
+
+    global::scriptEngine->queueScript(
+        "openspace.addScreenSpaceRenderable(" + browser + ");",
+        scripting::ScriptEngine::RemoteScripting::No
+    );
+
+    global::scriptEngine->queueScript(
+        "openspace.addScreenSpaceRenderable(" + target + ");",
+        scripting::ScriptEngine::RemoteScripting::No
+    );
+
+    global::scriptEngine->queueScript(
+        "openspace.skybrowser.addPairToSkyBrowserModule('" + idTarget + "','"
+        + idBrowser + "');",
+        scripting::ScriptEngine::RemoteScripting::No
+    );
+
+    global::scriptEngine->queueScript(
+        "openspace.skybrowser.setSelectedBrowser('" + idBrowser + "');",
+        scripting::ScriptEngine::RemoteScripting::No
+    );
+
+    return 0;
+}
+
+int removeTargetBrowserPair(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 1, "lua::removeTargetBrowserPair");
+    std::string id = ghoul::lua::value<std::string>(L, 1);
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+    TargetBrowserPair* found = module->getPair(id);
+    if (found) {
+        std::string browser = found->browserId();
+        std::string target = found->targetId();
+
+        module->removeTargetBrowserPair(id);
+
+        // Remove from engine
+        global::scriptEngine->queueScript(
+            "openspace.removeScreenSpaceRenderable('" + browser + "');",
+            scripting::ScriptEngine::RemoteScripting::Yes
+        );
+
+        global::scriptEngine->queueScript(
+            "openspace.removeScreenSpaceRenderable('" + target + "');",
+            scripting::ScriptEngine::RemoteScripting::Yes
+        );
+    }
+    
+    return 0;
+}
+
+int translateScreenSpaceRenderable(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 5, "lua::translateScreenSpaceRenderable");
+    auto [id, startX, startY, transX, transY] = 
+        ghoul::lua::values<std::string, float, float, float, float>(L);
+    
+    ScreenSpaceRenderable* renderable = global::renderEngine->screenSpaceRenderable(id);
+
+    if (renderable) {
```

Comment:
same concept here. Instead of checking if the renderable exists, check if it doesn't and return an error saying that it could not be found